### PR TITLE
adding javascript flag cleanup steps

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ incorporated additional steps to use javascript for flag cleanup. 
* Jira/GitHub Issue numbers (if any): \_FFM-10269
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 1]
```diff
@@ -32,7 +26,15 @@ Or
 
 - Flags that have been globally set to a single value for longer than 30 days.
 
-### Mark flags for cleanup
+#### Finding Potentially Stale Flags
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:29]:
```suggestion
#### Finding potentially stale flags
```
We generally use lc for headings except for proper names etc.

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 2]
```diff
@@ -32,7 +26,15 @@ Or
...
+
+- At the top of your Feature Flags overview screen, you will find a list of filters.
+- One of those filters is labeled **Potentially Stale Flags**.
+- Click this list to see which flags have been highlighted by Harness as potentially stale.
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:33]:
```suggestion
To see the flags identified by Harness as potentially stale, go to the top of the **Feature Flags Overview** and select the **Potentially Stale Flags** filter.
```
optional word-nerd sug.

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 3]
```diff
@@ -63,62 +65,111 @@ At this time, the following languages are supported for flag cleanup automation:
...
 - Kotlin
 - Go
+- Javascript 
+
+There are some slight differences when using Javascript for flag cleanup automation compared to the other languages listed. Have a read to learn more about the differences in the implementation of flag cleanup automation in Javascript. 
 
 ### Set up a flag cleanup pipeline
 
-In order to use flag cleanup automation you will need to [import a pipeline template](https://developer.harness.io/docs/platform/git-experience/import-a-template-from-git/).
+ 1. In order to use flag cleanup automation you will need to [import a pipeline template](https://developer.harness.io/docs/platform/git-experience/import-a-template-from-git/). The GitHub repo to import from is [here](https://github.com/harness/flag_cleanup/blob/unscripted/docs/pipelines/flag_cleanup_pipeline.yaml).
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:74]:
```suggestion
 1. To use flag cleanup automation,  you must [import a pipeline template](https://developer.harness.io/docs/platform/git-experience/import-a-template-from-git/). The GitHub repo to import from is [here](https://github.com/harness/flag_cleanup/blob/unscripted/docs/pipelines/flag_cleanup_pipeline.yaml). 
```

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 4]
```diff
@@ -63,62 +65,111 @@ At this time, the following languages are supported for flag cleanup automation:
...
 
 ### Set up a flag cleanup pipeline
 
-In order to use flag cleanup automation you will need to [import a pipeline template](https://developer.harness.io/docs/platform/git-experience/import-a-template-from-git/).
+ 1. In order to use flag cleanup automation you will need to [import a pipeline template](https://developer.harness.io/docs/platform/git-experience/import-a-template-from-git/). The GitHub repo to import from is [here](https://github.com/harness/flag_cleanup/blob/unscripted/docs/pipelines/flag_cleanup_pipeline.yaml). 
+- Note that the identifier and the name must match.
 
-- The URL to import from is https://github.com/harness/flag_cleanup/blob/unscripted/docs/pipelines/flag_cleanup_pipeline.yaml
-- Note that the identifier and the name must match
+ 2. To configure secrets, here are the steps to do this:
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:77]:
```suggestion
 2. To configure secrets:
```

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 5]
```diff
@@ -63,62 +65,111 @@ At this time, the following languages are supported for flag cleanup automation:
...
 
-<DocVideo src="https://www.youtube.com/embed/sSP1nxrBwxo?si=dGI7vBmio6pfhWnX" />
+ - A Github connector to connect to the repo to clone down the code to cleanup. The [demo repo is here](https://github.com/harness/flag_cleanup) and the branch to use is “unscripted”.
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:79]:
```suggestion
 - A Github connector to connect to the repo to clone the code to clean up. The [demo repo is here](https://github.com/harness/flag_cleanup) and the branch to use is “unscripted”.
```

File: docs/feature-flags/ff-creating-flag/manage-stale-flags.md [hunk 6]
```diff
@@ -63,62 +65,111 @@ At this time, the following languages are supported for flag cleanup automation:
...
+You can then navigate to the PR and see the changes it made.
 
-- A Docker connector to pull down the plugin image.
+<DocVideo src="https://www.youtube.com/embed/sSP1nxrBwxo?si=dGI7vBmio6pfhWnX" />
 
-![Step four of importing a pipeline template](./static/setting-up-cleanup-4.png)
+### Understanding the JavaScript Flag Cleanup Tool
 
-### Run the flag cleanup pipeline
+This guide helps you understand the `properties.json` file structure, its role, and how to write custom flag cleanup rules for JavaScript.
```
Comment (douglas-j-bothwell) [inline docs/feature-flags/ff-creating-flag/manage-stale-flags.md:123]:
```suggestion
This section describes the `properties.json` file structure, its role, and how to write custom flag cleanup rules for JavaScript.
```

**Review by douglas-j-bothwell — APPROVED** [review]
A few word-nerd sugs, otherwise LGTM

**Comment by jenniferopal** [comment]
> A few word-nerd sugs, otherwise LGTM

Thank you!

**Pull request merged**